homeassistant/components/tasmota/sensor.py

```diff
@@ -95,6 +97,7 @@
     SENSOR_PRESSUREATSEALEVEL: {DEVICE_CLASS: DEVICE_CLASS_PRESSURE},
     SENSOR_PROXIMITY: {ICON: "mdi:ruler"},
     SENSOR_REACTIVE_POWERUSAGE: {DEVICE_CLASS: DEVICE_CLASS_POWER},
+    SENSOR_STATUS_SIGNAL: {ICON: "mdi:signal"},
```

Comment:
What unit is the value in?

Reply:
It's in dB

Reply:
Yes, exactly.

Side note: It would be good if we don't return the unit of measurement directly from the library instance but use the home assistant common constants. The unit of measurement is part of the home assistant api.